### ISSUE TITLE
Link to Arc KDE no longer works

### ISSUE THREAD

**Issue by Shackelford-Arden** [varlesh/Nylas-Arc-Dark-Theme#5]
The link that's listed on the main page that is meant to direct users to the Arc KDE page sends users to a 404. I'd recommend fixing it :)

**Comment by ArchangeGabriel** [comment]
Yup, new link should be https://github.com/PapirusDevelopmentTeam/arc-kde.

**Comment by Shackelford-Arden** [comment]
@ArchangeGabriel That's it! Reminds me I need to go check those out haha.
